starters/next13-zustand-bulma/README.md

```diff
@@ -1,38 +1,117 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# next13-zustand-bulma Starter Kit
 
-## Getting Started
+This starter kit features [**Next.js 13+**](https://beta.nextjs.org/docs) (with the `appDir` and server components), [**Zustand**](https://zustand-demo.pmnd.rs/) for client-side state management, and [**Bulma**](https://bulma.io/) for styling.
 
-First, run the development server:
+##### How to use this Starter Kit:
 
 ```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
+npm create @this-dot/starter -- --kit next13-zustand-bulma
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+<sub>(use any of `npm` / `yarn` / `pnpm`)</sub>
+
+_[(...or use one of our alternative setup instructions)](SETUP.md)_
+
+<details>
+<summary>&nbsp;&nbsp;<b>Table of Contents</b></summary>
+
+- [next13-zustand-bulma starter kit](#next13-zustand-bulma-starter-kit)
+  - [Tech Stack](#tech-stack)
+    - [Included Tooling](#included-tooling)
+  - [Getting Started](#getting-started)
+  - [Available Commands](#available-commands)
+  - [Project Details](#project-details)
+    - [Kit Organization / Architecture](#kit-organization--architecture)
+    - [Styling and Theme](#styling-and-theme)
+    - [Font Loading](#font-loading)
+    - [Colocation in directories](#colocation-in-directories)
+  - [Demo Implementation](#demo-implementation)
+  - [Deployment](#deployment)
+
+</details>
+
+## Tech Stack
+
+- [Next.js v13.x](https://nextjs.org)
+- [React v18.x](https://reactjs.org)
+- [Zustand v4.x](https://react-query.tanstack.com/)
+- [Bulma](https://bulma.io/)
+
+#### Included Tooling
+
+- [Jest](https://jestjs.io/) - Test runner
+- [Typescript v4](https://www.typescriptlang.org/) - Type checking
+- [Storybook v7](https://storybook.js.org/) - Component sandbox and docs
+- [ESLint](https://eslint.org/) - Code linting
+- [Prettier](https://prettier.io/) - Code formatting
+
+## Getting Started
+
+- start by [creating a new Starter Kit](SETUP.md) project
```

Comment:
When I was running this locally, if you click on the url, the kit name was missing  

```
http://localhost:3000/kits/SETUP.md
```

Reply:
So, are you saying you think the SETUP.md should have the kit name, like:

---

# next13-zustand-bulma Starter Kit

## Setup (Installation)

### CLI (Recommended)

```bash
npm create @this-dot/starter -- --kit next13-zustand-bulma
```

---

I am going to assume that and make this change, but if that's not what you meant let me know.

Reply:
fixed in 2d02ec6

Reply:
fixed in 30bd0a4